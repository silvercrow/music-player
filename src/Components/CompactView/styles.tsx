import styled from 'styled-components';

interface Props {
  visible: boolean;
}

export const CompactBox = styled.div<Props>`
  display: ${(p) => (p.visible === true ? 'block' : 'none')};
  position: relative;
  width: 100%;
  height: 76px;
  bottom: 0;
  background-color: #3c4553;
  padding: 16px 20px;
  margin-top: auto;
  background-color: #5b6467;
  background-image: linear-gradient(315deg, #5b6467 0%, #8b939a 74%);
  border-radius: 10px;
`;
export const ExtendButton = styled.button`
  position: absolute;
  left: 42%;
  top: -20px;
  padding: 12px 0;
  border-radius: 50%;

  background-color: #5b6467;

  transition: all 0.3s ease;
  &:hover {
    top: -30px;
    transition: all 0.3s ease;
  }
  width: 40px;
  height: 40px;
  background-repeat: no-repeat;
  background-size: 20px;
  background-position: center;
  background-color: #ffcd00;
`;
export const Arrow = styled.span`
  position: absolute;
  left: 12%;
  top: 5px;
  height: 30px;
  width: 30px;
  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' stroke='%23fff' strokeLinecap='round' strokeLinejoin='round' strokeWidth='2' class='feather feather-chevron-up' width='' viewBox='0 0 24 24'%3E%3Cdefs/%3E%3Cpath d='M18 15l-6-6-6 6'/%3E%3C/svg%3E");
`;

export const Content = styled.div`
  display: flex;
  align-items: center;
`;
export const ArtCover = styled.div`
  img {
    border-radius: 10px;
    width: 32px;
    height: 32px;
    -o-object-fit: cover;
    object-fit: cover;
    margin-right: 16px;
  }
`;
export const InfoWrapper = styled.div``;
export const Title = styled.div`
  font-size: 12px;
  line-height: 18px;
  font-weight: 800;
  color: #fff;
`;
export const Subtitle = styled.div`
  font-size: 10px;
  color: #fff;
`;
export const ButtonsWrapper = styled.div`
  margin-left: auto;
  display: flex;
`;
export const StopButton = styled.button`
  background-color: #ffff;
  height: 15px;
  width: 15px;
  cursor: pointer;
  margin-right: 24px;
`;
export const NextButton = styled.div`
  color: #fff;
  background-position: center;
  background-repeat: no-repeat;
  background-size: contain;
`;
export const Progress = styled.div`
  display: flex;
  align-items: center;
  padding: 6px 0;
`;
export const CurrentTime = styled.div`
  display: inline-block;
  font-size: 8px;
  line-height: 12px;
  color: #fff;
`;
export const ProgressBar = styled.div`
  height: 2px;
  width: 100%;
  margin: 0 8px;
  background-color: #1f2128;
  border-radius: 2px;
  overflow: hidden;
`;
export const ProgressFill = styled.div`
  display: block;
  position: relative;
  width: 40%;
  height: 100%;
  background-color: #ffcd00;
  border-radius: 6px;
`;
export const TimeLeft = styled.div`
  display: inline-block;
  font-size: 8px;
  line-height: 12px;
  color: #fff;
`;
