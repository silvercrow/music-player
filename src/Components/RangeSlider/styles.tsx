import styled from 'styled-components';

const height = '16px';
const thumbHeight = 40;
const trackHeight = '16px';

// colours
const upperColor = '#46505F';
const lowerColor = '#FFCD00';
const thumbColor = '#FFFF';
const thumbHoverColor = '#FFFF';
const upperBackground = `linear-gradient(to bottom, ${upperColor}, ${upperColor}) 100% 50% / 100% ${trackHeight} no-repeat transparent`;
const lowerBackground = `linear-gradient(to bottom, ${lowerColor}, ${lowerColor}) 100% 50% / 100% ${trackHeight} no-repeat transparent`;

const makeLongShadow = (color: string, size: string) => {
  let i = 18;
  let shadow = `${i}px 0 0 ${size} ${color}`;
  for (; i < 706; i++) {
    shadow = `${shadow}, ${i}px 0 0 ${size} ${color}`;
  }
  return shadow;
};

export const Wrapper = styled.div`
  /* width:100%; */
`;
export const Range = styled.input`
  overflow: hidden;
  display: block;
  appearance: none;
  max-width: 700px;
  width: 100%;
  margin: 0;
  height: ${height};
  cursor: pointer;
  border-radius: 20px;

  &:focus {
    outline: none;
  }

  &::-webkit-slider-runnable-track {
    width: 100%;
    height: ${height};
    background: ${lowerBackground};
  }

  &::-webkit-slider-thumb {
    position: relative;
    appearance: none;
    height: ${thumbHeight}px;
    width: ${thumbHeight}px;
    background: ${thumbColor};
    border-radius: 100%;
    border: 0;
    top: 50%;
    transform: translateY(-50%);
    box-shadow: ${makeLongShadow(upperColor, '-10px')};
    transition: background-color 150ms;
  }

  &::-moz-range-track,
  &::-moz-range-progress {
    width: 100%;
    height: ${height};
    background: ${upperBackground};
  }

  &::-moz-range-progress {
    background: ${lowerBackground};
  }

  &::-moz-range-thumb {
    appearance: none;
    margin: 0;
    height: ${thumbHeight};
    width: ${thumbHeight};
    background: ${thumbColor};
    border-radius: 100%;
    border: 0;
    transition: background-color 150ms;
  }

  &::-ms-track {
    width: 100%;
    height: ${height};
    border: 0;
    /* color needed to hide track marks */
    color: transparent;
    background: transparent;
  }

  &::-ms-fill-lower {
    background: ${lowerBackground};
  }

  &::-ms-fill-upper {
    background: ${upperBackground};
  }

  &::-ms-thumb {
    appearance: none;
    height: ${thumbHeight};
    width: ${thumbHeight};
    background: ${thumbColor};
    border-radius: 100%;
    border: 0;
    transition: background-color 150ms;
    /* IE Edge thinks it can support -webkit prefixes */
    top: 0;
    margin: 0;
    box-shadow: none;
  }

  &:hover,
  &:focus {
    &::-webkit-slider-thumb {
      background-color: ${thumbHoverColor};
    }
    &::-moz-range-thumb {
      background-color: ${thumbHoverColor};
    }
    &::-ms-thumb {
      background-color: ${thumbHoverColor};
    }
  }
`;
