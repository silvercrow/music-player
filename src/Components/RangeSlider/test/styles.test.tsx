import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Wrapper, Range } from '../styles';

test('Wrapper', () => {
  const tree = renderer.create(<Wrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Range', () => {
  const tree = renderer.create(<Range />).toJSON();
  expect(tree).toMatchSnapshot();
});
