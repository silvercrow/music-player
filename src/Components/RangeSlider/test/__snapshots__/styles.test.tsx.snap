// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Range 1`] = `
.c0 {
  overflow: hidden;
  display: block;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  max-width: 700px;
  width: 100%;
  margin: 0;
  height: 16px;
  cursor: pointer;
  border-radius: 20px;
}

.c0:focus {
  outline: none;
}

.c0::-webkit-slider-runnable-track {
  width: 100%;
  height: 16px;
  background: linear-gradient(to bottom,#FFCD00,#FFCD00) 100% 50% / 100% 16px no-repeat transparent;
}

.c0::-webkit-slider-thumb {
  position: relative;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  height: 40px;
  width: 40px;
  background: #FFFF;
  border-radius: 100%;
  border: 0;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  box-shadow: 18px 0 0 -10px #46505F,18px 0 0 -10px #46505F,19px 0 0 -10px #46505F,20px 0 0 -10px #46505F,21px 0 0 -10px #46505F,22px 0 0 -10px #46505F,23px 0 0 -10px #46505F,24px 0 0 -10px #46505F,25px 0 0 -10px #46505F,26px 0 0 -10px #46505F,27px 0 0 -10px #46505F,28px 0 0 -10px #46505F,29px 0 0 -10px #46505F,30px 0 0 -10px #46505F,31px 0 0 -10px #46505F,32px 0 0 -10px #46505F,33px 0 0 -10px #46505F,34px 0 0 -10px #46505F,35px 0 0 -10px #46505F,36px 0 0 -10px #46505F,37px 0 0 -10px #46505F,38px 0 0 -10px #46505F,39px 0 0 -10px #46505F,40px 0 0 -10px #46505F,41px 0 0 -10px #46505F,42px 0 0 -10px #46505F,43px 0 0 -10px #46505F,44px 0 0 -10px #46505F,45px 0 0 -10px #46505F,46px 0 0 -10px #46505F,47px 0 0 -10px #46505F,48px 0 0 -10px #46505F,49px 0 0 -10px #46505F,50px 0 0 -10px #46505F,51px 0 0 -10px #46505F,52px 0 0 -10px #46505F,53px 0 0 -10px #46505F,54px 0 0 -10px #46505F,55px 0 0 -10px #46505F,56px 0 0 -10px #46505F,57px 0 0 -10px #46505F,58px 0 0 -10px #46505F,59px 0 0 -10px #46505F,60px 0 0 -10px #46505F,61px 0 0 -10px #46505F,62px 0 0 -10px #46505F,63px 0 0 -10px #46505F,64px 0 0 -10px #46505F,65px 0 0 -10px #46505F,66px 0 0 -10px #46505F,67px 0 0 -10px #46505F,68px 0 0 -10px #46505F,69px 0 0 -10px #46505F,70px 0 0 -10px #46505F,71px 0 0 -10px #46505F,72px 0 0 -10px #46505F,73px 0 0 -10px #46505F,74px 0 0 -10px #46505F,75px 0 0 -10px #46505F,76px 0 0 -10px #46505F,77px 0 0 -10px #46505F,78px 0 0 -10px #46505F,79px 0 0 -10px #46505F,80px 0 0 -10px #46505F,81px 0 0 -10px #46505F,82px 0 0 -10px #46505F,83px 0 0 -10px #46505F,84px 0 0 -10px #46505F,85px 0 0 -10px #46505F,86px 0 0 -10px #46505F,87px 0 0 -10px #46505F,88px 0 0 -10px #46505F,89px 0 0 -10px #46505F,90px 0 0 -10px #46505F,91px 0 0 -10px #46505F,92px 0 0 -10px #46505F,93px 0 0 -10px #46505F,94px 0 0 -10px #46505F,95px 0 0 -10px #46505F,96px 0 0 -10px #46505F,97px 0 0 -10px #46505F,98px 0 0 -10px #46505F,99px 0 0 -10px #46505F,100px 0 0 -10px #46505F,101px 0 0 -10px #46505F,102px 0 0 -10px #46505F,103px 0 0 -10px #46505F,104px 0 0 -10px #46505F,105px 0 0 -10px #46505F,106px 0 0 -10px #46505F,107px 0 0 -10px #46505F,108px 0 0 -10px #46505F,109px 0 0 -10px #46505F,110px 0 0 -10px #46505F,111px 0 0 -10px #46505F,112px 0 0 -10px #46505F,113px 0 0 -10px #46505F,114px 0 0 -10px #46505F,115px 0 0 -10px #46505F,116px 0 0 -10px #46505F,117px 0 0 -10px #46505F,118px 0 0 -10px #46505F,119px 0 0 -10px #46505F,120px 0 0 -10px #46505F,121px 0 0 -10px #46505F,122px 0 0 -10px #46505F,123px 0 0 -10px #46505F,124px 0 0 -10px #46505F,125px 0 0 -10px #46505F,126px 0 0 -10px #46505F,127px 0 0 -10px #46505F,128px 0 0 -10px #46505F,129px 0 0 -10px #46505F,130px 0 0 -10px #46505F,131px 0 0 -10px #46505F,132px 0 0 -10px #46505F,133px 0 0 -10px #46505F,134px 0 0 -10px #46505F,135px 0 0 -10px #46505F,136px 0 0 -10px #46505F,137px 0 0 -10px #46505F,138px 0 0 -10px #46505F,139px 0 0 -10px #46505F,140px 0 0 -10px #46505F,141px 0 0 -10px #46505F,142px 0 0 -10px #46505F,143px 0 0 -10px #46505F,144px 0 0 -10px #46505F,145px 0 0 -10px #46505F,146px 0 0 -10px #46505F,147px 0 0 -10px #46505F,148px 0 0 -10px #46505F,149px 0 0 -10px #46505F,150px 0 0 -10px #46505F,151px 0 0 -10px #46505F,152px 0 0 -10px #46505F,153px 0 0 -10px #46505F,154px 0 0 -10px #46505F,155px 0 0 -10px #46505F,156px 0 0 -10px #46505F,157px 0 0 -10px #46505F,158px 0 0 -10px #46505F,159px 0 0 -10px #46505F,160px 0 0 -10px #46505F,161px 0 0 -10px #46505F,162px 0 0 -10px #46505F,163px 0 0 -10px #46505F,164px 0 0 -10px #46505F,165px 0 0 -10px #46505F,166px 0 0 -10px #46505F,167px 0 0 -10px #46505F,168px 0 0 -10px #46505F,169px 0 0 -10px #46505F,170px 0 0 -10px #46505F,171px 0 0 -10px #46505F,172px 0 0 -10px #46505F,173px 0 0 -10px #46505F,174px 0 0 -10px #46505F,175px 0 0 -10px #46505F,176px 0 0 -10px #46505F,177px 0 0 -10px #46505F,178px 0 0 -10px #46505F,179px 0 0 -10px #46505F,180px 0 0 -10px #46505F,181px 0 0 -10px #46505F,182px 0 0 -10px #46505F,183px 0 0 -10px #46505F,184px 0 0 -10px #46505F,185px 0 0 -10px #46505F,186px 0 0 -10px #46505F,187px 0 0 -10px #46505F,188px 0 0 -10px #46505F,189px 0 0 -10px #46505F,190px 0 0 -10px #46505F,191px 0 0 -10px #46505F,192px 0 0 -10px #46505F,193px 0 0 -10px #46505F,194px 0 0 -10px #46505F,195px 0 0 -10px #46505F,196px 0 0 -10px #46505F,197px 0 0 -10px #46505F,198px 0 0 -10px #46505F,199px 0 0 -10px #46505F,200px 0 0 -10px #46505F,201px 0 0 -10px #46505F,202px 0 0 -10px #46505F,203px 0 0 -10px #46505F,204px 0 0 -10px #46505F,205px 0 0 -10px #46505F,206px 0 0 -10px #46505F,207px 0 0 -10px #46505F,208px 0 0 -10px #46505F,209px 0 0 -10px #46505F,210px 0 0 -10px #46505F,211px 0 0 -10px #46505F,212px 0 0 -10px #46505F,213px 0 0 -10px #46505F,214px 0 0 -10px #46505F,215px 0 0 -10px #46505F,216px 0 0 -10px #46505F,217px 0 0 -10px #46505F,218px 0 0 -10px #46505F,219px 0 0 -10px #46505F,220px 0 0 -10px #46505F,221px 0 0 -10px #46505F,222px 0 0 -10px #46505F,223px 0 0 -10px #46505F,224px 0 0 -10px #46505F,225px 0 0 -10px #46505F,226px 0 0 -10px #46505F,227px 0 0 -10px #46505F,228px 0 0 -10px #46505F,229px 0 0 -10px #46505F,230px 0 0 -10px #46505F,231px 0 0 -10px #46505F,232px 0 0 -10px #46505F,233px 0 0 -10px #46505F,234px 0 0 -10px #46505F,235px 0 0 -10px #46505F,236px 0 0 -10px #46505F,237px 0 0 -10px #46505F,238px 0 0 -10px #46505F,239px 0 0 -10px #46505F,240px 0 0 -10px #46505F,241px 0 0 -10px #46505F,242px 0 0 -10px #46505F,243px 0 0 -10px #46505F,244px 0 0 -10px #46505F,245px 0 0 -10px #46505F,246px 0 0 -10px #46505F,247px 0 0 -10px #46505F,248px 0 0 -10px #46505F,249px 0 0 -10px #46505F,250px 0 0 -10px #46505F,251px 0 0 -10px #46505F,252px 0 0 -10px #46505F,253px 0 0 -10px #46505F,254px 0 0 -10px #46505F,255px 0 0 -10px #46505F,256px 0 0 -10px #46505F,257px 0 0 -10px #46505F,258px 0 0 -10px #46505F,259px 0 0 -10px #46505F,260px 0 0 -10px #46505F,261px 0 0 -10px #46505F,262px 0 0 -10px #46505F,263px 0 0 -10px #46505F,264px 0 0 -10px #46505F,265px 0 0 -10px #46505F,266px 0 0 -10px #46505F,267px 0 0 -10px #46505F,268px 0 0 -10px #46505F,269px 0 0 -10px #46505F,270px 0 0 -10px #46505F,271px 0 0 -10px #46505F,272px 0 0 -10px #46505F,273px 0 0 -10px #46505F,274px 0 0 -10px #46505F,275px 0 0 -10px #46505F,276px 0 0 -10px #46505F,277px 0 0 -10px #46505F,278px 0 0 -10px #46505F,279px 0 0 -10px #46505F,280px 0 0 -10px #46505F,281px 0 0 -10px #46505F,282px 0 0 -10px #46505F,283px 0 0 -10px #46505F,284px 0 0 -10px #46505F,285px 0 0 -10px #46505F,286px 0 0 -10px #46505F,287px 0 0 -10px #46505F,288px 0 0 -10px #46505F,289px 0 0 -10px #46505F,290px 0 0 -10px #46505F,291px 0 0 -10px #46505F,292px 0 0 -10px #46505F,293px 0 0 -10px #46505F,294px 0 0 -10px #46505F,295px 0 0 -10px #46505F,296px 0 0 -10px #46505F,297px 0 0 -10px #46505F,298px 0 0 -10px #46505F,299px 0 0 -10px #46505F,300px 0 0 -10px #46505F,301px 0 0 -10px #46505F,302px 0 0 -10px #46505F,303px 0 0 -10px #46505F,304px 0 0 -10px #46505F,305px 0 0 -10px #46505F,306px 0 0 -10px #46505F,307px 0 0 -10px #46505F,308px 0 0 -10px #46505F,309px 0 0 -10px #46505F,310px 0 0 -10px #46505F,311px 0 0 -10px #46505F,312px 0 0 -10px #46505F,313px 0 0 -10px #46505F,314px 0 0 -10px #46505F,315px 0 0 -10px #46505F,316px 0 0 -10px #46505F,317px 0 0 -10px #46505F,318px 0 0 -10px #46505F,319px 0 0 -10px #46505F,320px 0 0 -10px #46505F,321px 0 0 -10px #46505F,322px 0 0 -10px #46505F,323px 0 0 -10px #46505F,324px 0 0 -10px #46505F,325px 0 0 -10px #46505F,326px 0 0 -10px #46505F,327px 0 0 -10px #46505F,328px 0 0 -10px #46505F,329px 0 0 -10px #46505F,330px 0 0 -10px #46505F,331px 0 0 -10px #46505F,332px 0 0 -10px #46505F,333px 0 0 -10px #46505F,334px 0 0 -10px #46505F,335px 0 0 -10px #46505F,336px 0 0 -10px #46505F,337px 0 0 -10px #46505F,338px 0 0 -10px #46505F,339px 0 0 -10px #46505F,340px 0 0 -10px #46505F,341px 0 0 -10px #46505F,342px 0 0 -10px #46505F,343px 0 0 -10px #46505F,344px 0 0 -10px #46505F,345px 0 0 -10px #46505F,346px 0 0 -10px #46505F,347px 0 0 -10px #46505F,348px 0 0 -10px #46505F,349px 0 0 -10px #46505F,350px 0 0 -10px #46505F,351px 0 0 -10px #46505F,352px 0 0 -10px #46505F,353px 0 0 -10px #46505F,354px 0 0 -10px #46505F,355px 0 0 -10px #46505F,356px 0 0 -10px #46505F,357px 0 0 -10px #46505F,358px 0 0 -10px #46505F,359px 0 0 -10px #46505F,360px 0 0 -10px #46505F,361px 0 0 -10px #46505F,362px 0 0 -10px #46505F,363px 0 0 -10px #46505F,364px 0 0 -10px #46505F,365px 0 0 -10px #46505F,366px 0 0 -10px #46505F,367px 0 0 -10px #46505F,368px 0 0 -10px #46505F,369px 0 0 -10px #46505F,370px 0 0 -10px #46505F,371px 0 0 -10px #46505F,372px 0 0 -10px #46505F,373px 0 0 -10px #46505F,374px 0 0 -10px #46505F,375px 0 0 -10px #46505F,376px 0 0 -10px #46505F,377px 0 0 -10px #46505F,378px 0 0 -10px #46505F,379px 0 0 -10px #46505F,380px 0 0 -10px #46505F,381px 0 0 -10px #46505F,382px 0 0 -10px #46505F,383px 0 0 -10px #46505F,384px 0 0 -10px #46505F,385px 0 0 -10px #46505F,386px 0 0 -10px #46505F,387px 0 0 -10px #46505F,388px 0 0 -10px #46505F,389px 0 0 -10px #46505F,390px 0 0 -10px #46505F,391px 0 0 -10px #46505F,392px 0 0 -10px #46505F,393px 0 0 -10px #46505F,394px 0 0 -10px #46505F,395px 0 0 -10px #46505F,396px 0 0 -10px #46505F,397px 0 0 -10px #46505F,398px 0 0 -10px #46505F,399px 0 0 -10px #46505F,400px 0 0 -10px #46505F,401px 0 0 -10px #46505F,402px 0 0 -10px #46505F,403px 0 0 -10px #46505F,404px 0 0 -10px #46505F,405px 0 0 -10px #46505F,406px 0 0 -10px #46505F,407px 0 0 -10px #46505F,408px 0 0 -10px #46505F,409px 0 0 -10px #46505F,410px 0 0 -10px #46505F,411px 0 0 -10px #46505F,412px 0 0 -10px #46505F,413px 0 0 -10px #46505F,414px 0 0 -10px #46505F,415px 0 0 -10px #46505F,416px 0 0 -10px #46505F,417px 0 0 -10px #46505F,418px 0 0 -10px #46505F,419px 0 0 -10px #46505F,420px 0 0 -10px #46505F,421px 0 0 -10px #46505F,422px 0 0 -10px #46505F,423px 0 0 -10px #46505F,424px 0 0 -10px #46505F,425px 0 0 -10px #46505F,426px 0 0 -10px #46505F,427px 0 0 -10px #46505F,428px 0 0 -10px #46505F,429px 0 0 -10px #46505F,430px 0 0 -10px #46505F,431px 0 0 -10px #46505F,432px 0 0 -10px #46505F,433px 0 0 -10px #46505F,434px 0 0 -10px #46505F,435px 0 0 -10px #46505F,436px 0 0 -10px #46505F,437px 0 0 -10px #46505F,438px 0 0 -10px #46505F,439px 0 0 -10px #46505F,440px 0 0 -10px #46505F,441px 0 0 -10px #46505F,442px 0 0 -10px #46505F,443px 0 0 -10px #46505F,444px 0 0 -10px #46505F,445px 0 0 -10px #46505F,446px 0 0 -10px #46505F,447px 0 0 -10px #46505F,448px 0 0 -10px #46505F,449px 0 0 -10px #46505F,450px 0 0 -10px #46505F,451px 0 0 -10px #46505F,452px 0 0 -10px #46505F,453px 0 0 -10px #46505F,454px 0 0 -10px #46505F,455px 0 0 -10px #46505F,456px 0 0 -10px #46505F,457px 0 0 -10px #46505F,458px 0 0 -10px #46505F,459px 0 0 -10px #46505F,460px 0 0 -10px #46505F,461px 0 0 -10px #46505F,462px 0 0 -10px #46505F,463px 0 0 -10px #46505F,464px 0 0 -10px #46505F,465px 0 0 -10px #46505F,466px 0 0 -10px #46505F,467px 0 0 -10px #46505F,468px 0 0 -10px #46505F,469px 0 0 -10px #46505F,470px 0 0 -10px #46505F,471px 0 0 -10px #46505F,472px 0 0 -10px #46505F,473px 0 0 -10px #46505F,474px 0 0 -10px #46505F,475px 0 0 -10px #46505F,476px 0 0 -10px #46505F,477px 0 0 -10px #46505F,478px 0 0 -10px #46505F,479px 0 0 -10px #46505F,480px 0 0 -10px #46505F,481px 0 0 -10px #46505F,482px 0 0 -10px #46505F,483px 0 0 -10px #46505F,484px 0 0 -10px #46505F,485px 0 0 -10px #46505F,486px 0 0 -10px #46505F,487px 0 0 -10px #46505F,488px 0 0 -10px #46505F,489px 0 0 -10px #46505F,490px 0 0 -10px #46505F,491px 0 0 -10px #46505F,492px 0 0 -10px #46505F,493px 0 0 -10px #46505F,494px 0 0 -10px #46505F,495px 0 0 -10px #46505F,496px 0 0 -10px #46505F,497px 0 0 -10px #46505F,498px 0 0 -10px #46505F,499px 0 0 -10px #46505F,500px 0 0 -10px #46505F,501px 0 0 -10px #46505F,502px 0 0 -10px #46505F,503px 0 0 -10px #46505F,504px 0 0 -10px #46505F,505px 0 0 -10px #46505F,506px 0 0 -10px #46505F,507px 0 0 -10px #46505F,508px 0 0 -10px #46505F,509px 0 0 -10px #46505F,510px 0 0 -10px #46505F,511px 0 0 -10px #46505F,512px 0 0 -10px #46505F,513px 0 0 -10px #46505F,514px 0 0 -10px #46505F,515px 0 0 -10px #46505F,516px 0 0 -10px #46505F,517px 0 0 -10px #46505F,518px 0 0 -10px #46505F,519px 0 0 -10px #46505F,520px 0 0 -10px #46505F,521px 0 0 -10px #46505F,522px 0 0 -10px #46505F,523px 0 0 -10px #46505F,524px 0 0 -10px #46505F,525px 0 0 -10px #46505F,526px 0 0 -10px #46505F,527px 0 0 -10px #46505F,528px 0 0 -10px #46505F,529px 0 0 -10px #46505F,530px 0 0 -10px #46505F,531px 0 0 -10px #46505F,532px 0 0 -10px #46505F,533px 0 0 -10px #46505F,534px 0 0 -10px #46505F,535px 0 0 -10px #46505F,536px 0 0 -10px #46505F,537px 0 0 -10px #46505F,538px 0 0 -10px #46505F,539px 0 0 -10px #46505F,540px 0 0 -10px #46505F,541px 0 0 -10px #46505F,542px 0 0 -10px #46505F,543px 0 0 -10px #46505F,544px 0 0 -10px #46505F,545px 0 0 -10px #46505F,546px 0 0 -10px #46505F,547px 0 0 -10px #46505F,548px 0 0 -10px #46505F,549px 0 0 -10px #46505F,550px 0 0 -10px #46505F,551px 0 0 -10px #46505F,552px 0 0 -10px #46505F,553px 0 0 -10px #46505F,554px 0 0 -10px #46505F,555px 0 0 -10px #46505F,556px 0 0 -10px #46505F,557px 0 0 -10px #46505F,558px 0 0 -10px #46505F,559px 0 0 -10px #46505F,560px 0 0 -10px #46505F,561px 0 0 -10px #46505F,562px 0 0 -10px #46505F,563px 0 0 -10px #46505F,564px 0 0 -10px #46505F,565px 0 0 -10px #46505F,566px 0 0 -10px #46505F,567px 0 0 -10px #46505F,568px 0 0 -10px #46505F,569px 0 0 -10px #46505F,570px 0 0 -10px #46505F,571px 0 0 -10px #46505F,572px 0 0 -10px #46505F,573px 0 0 -10px #46505F,574px 0 0 -10px #46505F,575px 0 0 -10px #46505F,576px 0 0 -10px #46505F,577px 0 0 -10px #46505F,578px 0 0 -10px #46505F,579px 0 0 -10px #46505F,580px 0 0 -10px #46505F,581px 0 0 -10px #46505F,582px 0 0 -10px #46505F,583px 0 0 -10px #46505F,584px 0 0 -10px #46505F,585px 0 0 -10px #46505F,586px 0 0 -10px #46505F,587px 0 0 -10px #46505F,588px 0 0 -10px #46505F,589px 0 0 -10px #46505F,590px 0 0 -10px #46505F,591px 0 0 -10px #46505F,592px 0 0 -10px #46505F,593px 0 0 -10px #46505F,594px 0 0 -10px #46505F,595px 0 0 -10px #46505F,596px 0 0 -10px #46505F,597px 0 0 -10px #46505F,598px 0 0 -10px #46505F,599px 0 0 -10px #46505F,600px 0 0 -10px #46505F,601px 0 0 -10px #46505F,602px 0 0 -10px #46505F,603px 0 0 -10px #46505F,604px 0 0 -10px #46505F,605px 0 0 -10px #46505F,606px 0 0 -10px #46505F,607px 0 0 -10px #46505F,608px 0 0 -10px #46505F,609px 0 0 -10px #46505F,610px 0 0 -10px #46505F,611px 0 0 -10px #46505F,612px 0 0 -10px #46505F,613px 0 0 -10px #46505F,614px 0 0 -10px #46505F,615px 0 0 -10px #46505F,616px 0 0 -10px #46505F,617px 0 0 -10px #46505F,618px 0 0 -10px #46505F,619px 0 0 -10px #46505F,620px 0 0 -10px #46505F,621px 0 0 -10px #46505F,622px 0 0 -10px #46505F,623px 0 0 -10px #46505F,624px 0 0 -10px #46505F,625px 0 0 -10px #46505F,626px 0 0 -10px #46505F,627px 0 0 -10px #46505F,628px 0 0 -10px #46505F,629px 0 0 -10px #46505F,630px 0 0 -10px #46505F,631px 0 0 -10px #46505F,632px 0 0 -10px #46505F,633px 0 0 -10px #46505F,634px 0 0 -10px #46505F,635px 0 0 -10px #46505F,636px 0 0 -10px #46505F,637px 0 0 -10px #46505F,638px 0 0 -10px #46505F,639px 0 0 -10px #46505F,640px 0 0 -10px #46505F,641px 0 0 -10px #46505F,642px 0 0 -10px #46505F,643px 0 0 -10px #46505F,644px 0 0 -10px #46505F,645px 0 0 -10px #46505F,646px 0 0 -10px #46505F,647px 0 0 -10px #46505F,648px 0 0 -10px #46505F,649px 0 0 -10px #46505F,650px 0 0 -10px #46505F,651px 0 0 -10px #46505F,652px 0 0 -10px #46505F,653px 0 0 -10px #46505F,654px 0 0 -10px #46505F,655px 0 0 -10px #46505F,656px 0 0 -10px #46505F,657px 0 0 -10px #46505F,658px 0 0 -10px #46505F,659px 0 0 -10px #46505F,660px 0 0 -10px #46505F,661px 0 0 -10px #46505F,662px 0 0 -10px #46505F,663px 0 0 -10px #46505F,664px 0 0 -10px #46505F,665px 0 0 -10px #46505F,666px 0 0 -10px #46505F,667px 0 0 -10px #46505F,668px 0 0 -10px #46505F,669px 0 0 -10px #46505F,670px 0 0 -10px #46505F,671px 0 0 -10px #46505F,672px 0 0 -10px #46505F,673px 0 0 -10px #46505F,674px 0 0 -10px #46505F,675px 0 0 -10px #46505F,676px 0 0 -10px #46505F,677px 0 0 -10px #46505F,678px 0 0 -10px #46505F,679px 0 0 -10px #46505F,680px 0 0 -10px #46505F,681px 0 0 -10px #46505F,682px 0 0 -10px #46505F,683px 0 0 -10px #46505F,684px 0 0 -10px #46505F,685px 0 0 -10px #46505F,686px 0 0 -10px #46505F,687px 0 0 -10px #46505F,688px 0 0 -10px #46505F,689px 0 0 -10px #46505F,690px 0 0 -10px #46505F,691px 0 0 -10px #46505F,692px 0 0 -10px #46505F,693px 0 0 -10px #46505F,694px 0 0 -10px #46505F,695px 0 0 -10px #46505F,696px 0 0 -10px #46505F,697px 0 0 -10px #46505F,698px 0 0 -10px #46505F,699px 0 0 -10px #46505F,700px 0 0 -10px #46505F,701px 0 0 -10px #46505F,702px 0 0 -10px #46505F,703px 0 0 -10px #46505F,704px 0 0 -10px #46505F,705px 0 0 -10px #46505F;
  -webkit-transition: background-color 150ms;
  transition: background-color 150ms;
}

.c0::-moz-range-track,
.c0::-moz-range-progress {
  width: 100%;
  height: 16px;
  background: linear-gradient(to bottom,#46505F,#46505F) 100% 50% / 100% 16px no-repeat transparent;
}

.c0::-moz-range-progress {
  background: linear-gradient(to bottom,#FFCD00,#FFCD00) 100% 50% / 100% 16px no-repeat transparent;
}

.c0::-moz-range-thumb {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  margin: 0;
  height: 40;
  width: 40;
  background: #FFFF;
  border-radius: 100%;
  border: 0;
  -webkit-transition: background-color 150ms;
  transition: background-color 150ms;
}

.c0::-ms-track {
  width: 100%;
  height: 16px;
  border: 0;
  color: transparent;
  background: transparent;
}

.c0::-ms-fill-lower {
  background: linear-gradient(to bottom,#FFCD00,#FFCD00) 100% 50% / 100% 16px no-repeat transparent;
}

.c0::-ms-fill-upper {
  background: linear-gradient(to bottom,#46505F,#46505F) 100% 50% / 100% 16px no-repeat transparent;
}

.c0::-ms-thumb {
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  height: 40;
  width: 40;
  background: #FFFF;
  border-radius: 100%;
  border: 0;
  -webkit-transition: background-color 150ms;
  transition: background-color 150ms;
  top: 0;
  margin: 0;
  box-shadow: none;
}

.c0:hover::-webkit-slider-thumb,
.c0:focus::-webkit-slider-thumb {
  background-color: #FFFF;
}

.c0:hover::-moz-range-thumb,
.c0:focus::-moz-range-thumb {
  background-color: #FFFF;
}

.c0:hover::-ms-thumb,
.c0:focus::-ms-thumb {
  background-color: #FFFF;
}

<input
  className="c0"
/>
`;

exports[`Wrapper 1`] = `
<div
  className=""
/>
`;
